### ISSUE TITLE
Unable to replicate result

### ISSUE THREAD

**Issue by deancorleone00** [zcakhaa/DeepLOB-Deep-Convolutional-Neural-Networks-for-Limit-Order-Books#7]
Thanks for the awesome work. I can't replicate the result presented in paper (around 75~85 for both accuracy and F1 score depending on the horizon) following your notebook, is there anything I am missing?

The F1 score I get
Macro: 0.5184215198116572
Micro: 0.5824873824271622
Weighted: 0.5740045835109294

Accuracy is similar to the output of your notebook (~0.58).

**Comment by zcakhaa** [comment]
If you check run_train_tensorflow-version1.ipynb, you will see the results that go to 75%.
You just need to run more epochs on tensorflow2. The training between tensorflow 1 and 2 seems to be slightly different.


________________________________
